### PR TITLE
Use "git" CLI for cloning.

### PR DESCRIPTION
**Pull request by oliverchang**
This is much faster than libgit2's clone for large repos.

### CODE REVIEW

File: lib/osv/repos.py [hunk 1]
```diff
@@ -24,16 +25,50 @@
 RETRY_SLEEP_SECONDS = 5
 
 
+class GitRemoteCallback(pygit2.RemoteCallbacks):
```
Comment (inferno-chromium) [inline lib/osv/repos.py:28]: where is GitRemoteCallback used, i dont see any callsite changes after move.

File: docker/worker/worker_test.py [hunk 2]
```diff
@@ -834,7 +834,7 @@ def setUp(self):
     self.maxDiff = None
     tests.reset_emulator()
 
-    self.original_clone = pygit2.clone_repository
+    self.original_clone = osv.clone
```
Comment (inferno-chromium) [inline docker/worker/worker_test.py:837]: osv.repos.clone ?

**Review by inferno-chromium — APPROVED** [review]

File: lib/osv/repos.py (hunk 1 above)
Comment (oliverchang) [reply lib/osv/repos.py:28]: In worker.py and importer.py, by users of the osv lib.

File: docker/worker/worker_test.py (hunk 2 above)
Comment (oliverchang) [reply docker/worker/worker_test.py:837]: osv.repos.clone is the internal import path. osv.clone is the external one.

**Comment by jonathanmetzman** [comment]
> This is much faster than libgit2's clone for large repos.

Sad this is the case.
LGTM

**Pull request merged**